test/unittests/skip_space_iterator_test.cpp

```diff
@@ -0,0 +1,84 @@
+// EVMC: Ethereum Client-VM Connector API.
+// Copyright 2022 The EVMC Authors.
+// Licensed under the Apache License, Version 2.0.
+
+#include <tools/evmc/skip_space_iterator.hpp>
+#include <gtest/gtest.h>
+#include <cctype>
+
+using evmc::skip_space_iterator;
+
+namespace
+{
+std::string remove_space(std::string_view in)
+{
+    // Copy input to additional buffer. This helps with out-of-buffer reads detection by sanitizers.
+    const auto in_buffer = std::make_unique<char[]>(in.size());
```

Comment:
Maybe for a test just simpler to use `std::string in_buffer(in)`

Reply:
Could be `vector<char>` maybe. Just doing this in 4 lines feels like unnecessary complication.